scripts/build_awssdk.bat

```diff
@@ -3,7 +3,7 @@
 :: GitHub repo: https://github.com/aws/aws-sdk-cpp.git
 ::
 @echo off
-set aws_version=1.3.50.1
+set aws_version=1.3.50.2

```

Comment:
why we need this change for this issue?

Reply:
This is for version management for third-parties. The artifacts of third-parties will be uploaded to S3 storage only when the version is new and not found on S3. ODBC driver or other Snowflake application needs those third-parties will pick up them from there.